docs/csv/example_files/ballot_measure_contest.txt

```diff
@@ -1,2 +1 @@
-con_statement,effect_of_abstain,full_text,info_uri,passage_threshold,pro_statement,summary_text,type,other_type,id
-This is no good.,Nothing will happen.,,,,Everything will be great.,,referendum,,,,bmc30001
+abbreviation,Contest,ballot_measure_selection_ids,ballot_sub_title,ballot_title,elecoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation,con_statement,effect_of_abstain,full_text,info_uri,passage_threshold,pro_statement,summary_text,type,other_type,id
```

Comment:
What was the motivation behind removing the example lines from the example_files files?